detox/src/ios/earlgreyapi/EarlGrey.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
We can probably throw this away, as this class is empty. Everything is in EarlGreyImpl